src/app/space/create/deployments/services/deployments.service.spec.ts

```diff
@@ -133,10 +133,10 @@ describe('DeploymentsService', () => {
   });
 
   describe('#getEnvironments', () => {
-    it('should publish faked environments', (done: DoneFn) => {
+    it('should publish faked, filtered and sorted environments', (done: DoneFn) => {
       const expectedResponse = {
         data: [
-          { name: 'stage' }, { name: 'run' }, { name: 'test' }
+          { name: 'stage' }, { name: 'run' }
```

Comment:
`expectedResponse` is a bit of a misleading name - it's both the response sent back by the mock HTTP backend, and in most tests, also contains the expected result data.

In this case the data array should still contain a 'test' member, and 'stage' and 'run' should not be in the desired order.